backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/transformer/EnsuingDecisionTransformer.java

```diff
@@ -32,8 +33,8 @@ public static EnsuingDecisionDTO transformToDTO(EnsuingDecision ensuingDecision)
         .date(ensuingDecision.getDecisionDate())
         .documentNumber(ensuingDecision.getDocumentNumber())
         .documentType(getDocumentTypeFromDomain(ensuingDecision.getDocumentType()))
-        .fileNumber(ensuingDecision.getFileNumber())
-        .note(ensuingDecision.getNote())
+        .fileNumber(StringUtils.normalizeSpace(ensuingDecision.getFileNumber()))
+        .note(StringUtils.normalizeSpace(ensuingDecision.getNote()))
```

Comment:
Not sure what this note is about but do we really have to normalize this?

Reply:
Same here. It's the "Vermerk" in "Nachgehende Entscheidung" and should probably not be normalized. I will change that back